docs/conf.py

```diff
@@ -39,6 +39,8 @@
 html_split_index = False
 html_short_title = f'{project}-{version}'
 
+linkcheck_anchors_ignore_for_url = [r'https?://(www\.)?github\.com/.*']
```

Comment:
Let's add that beginning of the match marker too.
```suggestion
linkcheck_anchors_ignore_for_url = [
    r'^https?://(www\.)?github\.com/.*',
]
```